Kekka/Classes/Typealiases.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Typealiases.swift
+//  Kekka
+//
+//  Created by Vijaya Prakash Kandel on 22.03.19.
+//  Copyright © 2019 com.kandelvijaya. All rights reserved.
+//
+
+import Foundation
+
+/// Useful when declaring completion.
+public typealias ClosedBlock = () -> Void
```

Comment:
I'm not a big fan of this naming. You might want to take inspiration from elsewhere, e.g. https://docs.flutter.io/flutter/dart-ui/VoidCallback.html. Something with `Void` is more generally understandable